pkg/cmd/ctlstore/main.go

```diff
@@ -275,10 +285,24 @@ func supervisor(ctx context.Context, args []string) {
 		if cliCfg.Shadow {
 			shadow = "true"
 		}
+
+		var promHandler *prometheus.Handler
+		if len(reflectorConfig.MetricsBind) > 0 {
+			promHandler = &prometheus.Handler{}
+
+			http.Handle("/metrics", promHandler)
+
+			go func() {
+				events.Log("Serving Prometheus metrics on %s", reflectorConfig.MetricsBind)
+				http.ListenAndServe(reflectorConfig.MetricsBind, nil)
```

Comment:
Would recommend logging the error return here.

Reply:
done